src/require-hook.js

```diff
@@ -25,6 +27,30 @@ if (process.type === 'renderer') {
       window.__hot.forEach(fn => fn());
     });
   }
+
+  if (stylesheetReload) {
+    electron.ipcRenderer.on('__electron-compile__stylesheet_reload', (e, path) => {
+      /*if (path.match(/.(jpg|jpeg|png|gif)$/i)) {
+        let images = document.getElementsByTagName('img');
+
+        for (let img of images) {
+          let uri = img.src
+          if (uri.includes(path)) {
+            img.src = img.src; // trigger an update
+          }
+        }
+      } else {*/
```

Comment:
I was going to make it so we could hot reload assets like images, but I'm not sure how these are treated by the dependency finder. If it's relatively plug and play I'll finish that up otherwise we can remove it.